include/nbl/ui/CWindowWin32.h

```diff
@@ -1,6 +1,6 @@
 #ifndef __C_WINDOW_WIN32_H_INCLUDED__
 #define __C_WINDOW_WIN32_H_INCLUDED__
-
+#ifdef _NBL_PLATFORM_WINDOWS_
```

Comment:
1) you have 2x `ifdef _NBL_PLATFORM_WINDOWS_`
2) you have include IWindowWin32 before the ifdef to have even a chance for _NBL_PLATFORM_WINDOWS_ to be defined (it includes compile_config.h indirectly)